History.md

```diff
@@ -1,3 +1,9 @@
+unreleased
```

Comment:
Hi @tunniclm, you want to look only at the commits made on the 5.0 branch, not al the commits merged into there. You can use the git commands to walk only the left-hand of merges to just see commits that were committed against a branch rather than looking at the really confusing GitHub smashed view.

When you open each one of those commits, GitHub will even show what branch that commit was against in the header.